aggregator/src/aggregator/aggregation_job_driver.rs

```diff
@@ -68,16 +68,16 @@ impl AggregationJobDriver {
             .u64_counter("janus_job_cancellations")
             .with_description("Count of cancelled jobs.")
             .init();
-        job_cancel_counter.add(&Context::current(), 0, &[]);
+        job_cancel_counter.add(0, &[]);
 
         let job_retry_counter = meter
             .u64_counter("janus_job_retries")
             .with_description("Count of retried job steps.")
             .init();
-        job_retry_counter.add(&Context::current(), 0, &[]);
+        job_retry_counter.add(0, &[]);
 
         let http_request_duration_histogram = meter
-            .f64_histogram("janus_http_request_duration_seconds")
+            .f64_histogram("janus_http_request_duration")
```

Comment:
Was changing the histogram name intentional?

Reply:
Yes, see #1657: we need to drop unit names from our metrics since they are now added (at least for Prometheus, lol).